recipes-core/components/rh-sinkvita49_3.1.0.bb

```diff
@@ -2,7 +2,12 @@ DESCRIPTION = "REDHAWK SinkVITA49"
 
 DEPENDS = "bulkiointerfaces rh-vita49"
 RDEPENDS_${PN} = "bulkiointerfaces rh-vita49"
-
+LIC_FILES_CHSUM = "0"
```

Comment:
This variable is invalid (and unnecessary since the checksum gets set in `recipes-core/core-framework/core-framework.inc`).  Please remove.

Reply:
Got it.